README.md

```diff
@@ -1,10 +1,132 @@
-### sneaker-portfolio
-- `docker-compose up`
+# Sneaker Portfolio  
+<img src = readme-media/gifs/portfolio.gif > </img>
 
-### sneaker-server
-- `poetry run uvicorn api.main:app` to run Backend
 
-### sneaker-fronted
-- `npm install`
-- `npm run build` for building static website for FastAPI
-- `npm start`
+  
+
+# [Check out the Website here!](https://nsli.me)
```

Comment:
Probably should be above the gif, because its a sea of blue here, one may not realize its a link or read it. 

And maybe

[Website] <- Check it out

So its clear there is a website Link